meshtastic/tests/test_ota.py

```
@@ -231,7 +231,178 @@ def test_esp32_wifi_ota_update_success(mock_socket_class: MagicMock) -> None:
+        ota = ESP32WiFiOTA(temp_file, "192.168.1.1")
+        assert ota._size == len(original_data)
+        assert ota.hash_hex() == hashlib.sha256(original_data).hexdigest()
+        replacement_data = b"B" * ota._size
+        with open(temp_file, "wb") as fw:
+            fw.write(replacement_data)
+
+        with patch.object(ota, "_read_line") as mock_read_line:
+            mock_read_line.side_effect = ["OK", "OK"]
+            ota.update()
+
+        start_cmd = f"OTA {len(original_data)} {ota.hash_hex()}\n".encode("utf-8")
```

Ruff (0.15.5)

[warning] Unnecessary UTF-8 `encoding` argument to `encode`

Remove unnecessary `encoding` argument

(UP012)